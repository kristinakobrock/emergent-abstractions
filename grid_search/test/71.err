  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 27.46it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 24.68it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 25.01it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 23.83it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 24.17it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 22.81it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 23.54it/s] 48%|############################################1                                               | 24/50 [00:00<00:01, 23.98it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 25.69it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 26.74it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 25.30it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 25.32it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 24.24it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 24.58it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 25.84it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 24.64it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.02it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 29.40it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 26.78it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 28.27it/s]100%|############################################################################################| 13/13 [00:00<00:00, 26.96it/s]100%|############################################################################################| 13/13 [00:00<00:00, 27.31it/s]
