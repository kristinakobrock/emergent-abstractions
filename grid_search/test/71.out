# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7909213304519653, "acc": 0.5427851676940918, "length": 3.28568959236145, "mode": "train", "epoch": 1}
{"loss": 0.7299597263336182, "acc": 0.4083983898162842, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6842251420021057, "acc": 0.5839152932167053, "length": 1.8004693984985352, "mode": "train", "epoch": 2}
{"loss": 0.661261260509491, "acc": 0.630664050579071, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6668962240219116, "acc": 0.6259903311729431, "length": 1.0830204486846924, "mode": "train", "epoch": 3}
{"loss": 0.6644352674484253, "acc": 0.630664050579071, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6617359519004822, "acc": 0.6277307868003845, "length": 1.0215030908584595, "mode": "train", "epoch": 4}
{"loss": 0.6609737873077393, "acc": 0.630664050579071, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.660895824432373, "acc": 0.6278480291366577, "length": 1.0133085250854492, "mode": "train", "epoch": 5}
{"loss": 0.6614863872528076, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6596836447715759, "acc": 0.6278477907180786, "length": 1.008468747138977, "mode": "train", "epoch": 6}
{"loss": 0.6592411994934082, "acc": 0.630664050579071, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6597195863723755, "acc": 0.6276759505271912, "length": 1.0061378479003906, "mode": "train", "epoch": 7}
{"loss": 0.6593380570411682, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6592277884483337, "acc": 0.6276761293411255, "length": 1.004929780960083, "mode": "train", "epoch": 8}
{"loss": 0.6590124368667603, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6587302088737488, "acc": 0.627675473690033, "length": 1.0044474601745605, "mode": "train", "epoch": 9}
{"loss": 0.6588155031204224, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6589633226394653, "acc": 0.6276753544807434, "length": 1.0038424730300903, "mode": "train", "epoch": 10}
{"loss": 0.6589329242706299, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6584907174110413, "acc": 0.6277329325675964, "length": 1.0035791397094727, "mode": "train", "epoch": 11}
{"loss": 0.6597738265991211, "acc": 0.630664050579071, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6583917737007141, "acc": 0.6277332305908203, "length": 1.0031538009643555, "mode": "train", "epoch": 12}
{"loss": 0.6593873500823975, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6582704186439514, "acc": 0.6277907490730286, "length": 1.0029720067977905, "mode": "train", "epoch": 13}
{"loss": 0.6586639285087585, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6582711935043335, "acc": 0.6277903914451599, "length": 1.002654790878296, "mode": "train", "epoch": 14}
{"loss": 0.6591119170188904, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.657690703868866, "acc": 0.6278477907180786, "length": 1.0023490190505981, "mode": "train", "epoch": 15}
{"loss": 0.6590422987937927, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6579221487045288, "acc": 0.6279057860374451, "length": 1.002152442932129, "mode": "train", "epoch": 16}
{"loss": 0.6593348979949951, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6578806042671204, "acc": 0.6278481483459473, "length": 1.0019447803497314, "mode": "train", "epoch": 17}
{"loss": 0.6590552926063538, "acc": 0.6304687857627869, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6577191948890686, "acc": 0.6279060244560242, "length": 1.001853585243225, "mode": "train", "epoch": 18}
{"loss": 0.6598844528198242, "acc": 0.630664050579071, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6577085256576538, "acc": 0.627906084060669, "length": 1.0015984773635864, "mode": "train", "epoch": 19}
{"loss": 0.6594647765159607, "acc": 0.6304687857627869, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.6574841737747192, "acc": 0.6280202865600586, "length": 1.001517415046692, "mode": "train", "epoch": 20}
{"loss": 0.6601733565330505, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6580296754837036, "acc": 0.6280773878097534, "length": 1.001394510269165, "mode": "train", "epoch": 21}
{"loss": 0.6591415405273438, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.6573036909103394, "acc": 0.6281922459602356, "length": 1.0012099742889404, "mode": "train", "epoch": 22}
{"loss": 0.6600663661956787, "acc": 0.6304687857627869, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.6580719351768494, "acc": 0.6279048323631287, "length": 1.001142978668213, "mode": "train", "epoch": 23}
{"loss": 0.659751296043396, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6572155356407166, "acc": 0.6279630064964294, "length": 1.0011162757873535, "mode": "train", "epoch": 24}
{"loss": 0.6604839563369751, "acc": 0.6312499642372131, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.6572866439819336, "acc": 0.6283077001571655, "length": 1.000880479812622, "mode": "train", "epoch": 25}
{"loss": 0.6591460108757019, "acc": 0.6308594346046448, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.6564934849739075, "acc": 0.6282509565353394, "length": 1.0008407831192017, "mode": "train", "epoch": 26}
{"loss": 0.6602555513381958, "acc": 0.6308594346046448, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6570184826850891, "acc": 0.6283082365989685, "length": 1.0007811784744263, "mode": "train", "epoch": 27}
{"loss": 0.6595134735107422, "acc": 0.6308594346046448, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6566426157951355, "acc": 0.6281932592391968, "length": 1.0006972551345825, "mode": "train", "epoch": 28}
{"loss": 0.6596858501434326, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.6572728753089905, "acc": 0.6280781030654907, "length": 1.0006870031356812, "mode": "train", "epoch": 29}
{"loss": 0.6599217057228088, "acc": 0.6308594346046448, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.6568466424942017, "acc": 0.6282501220703125, "length": 1.0005799531936646, "mode": "train", "epoch": 30}
{"loss": 0.6607799530029297, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 30}
{"loss": 0.6566382050514221, "acc": 0.6280781626701355, "length": 1.000617265701294, "mode": "train", "epoch": 31}
{"loss": 0.6596026420593262, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 31}
{"loss": 0.656826913356781, "acc": 0.6283653378486633, "length": 1.0005075931549072, "mode": "train", "epoch": 32}
{"loss": 0.6605263352394104, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 32}
{"loss": 0.65667724609375, "acc": 0.628308117389679, "length": 1.0004186630249023, "mode": "train", "epoch": 33}
{"loss": 0.6608641147613525, "acc": 0.6302734017372131, "length": 1.0, "mode": "test", "epoch": 33}
{"loss": 0.6568092703819275, "acc": 0.6283080577850342, "length": 1.0003982782363892, "mode": "train", "epoch": 34}
{"loss": 0.6600340604782104, "acc": 0.6304686665534973, "length": 1.0, "mode": "test", "epoch": 34}
{"loss": 0.6565584540367126, "acc": 0.6282503008842468, "length": 1.0003557205200195, "mode": "train", "epoch": 35}
{"loss": 0.6601176261901855, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 35}
{"loss": 0.6567102074623108, "acc": 0.6282505393028259, "length": 1.000330924987793, "mode": "train", "epoch": 36}
{"loss": 0.6603000164031982, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 36}
{"loss": 0.6563645005226135, "acc": 0.6283655166625977, "length": 1.0003454685211182, "mode": "train", "epoch": 37}
{"loss": 0.6599213480949402, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 37}
{"loss": 0.6563234925270081, "acc": 0.628193199634552, "length": 1.0002936124801636, "mode": "train", "epoch": 38}
{"loss": 0.6609835624694824, "acc": 0.6304687857627869, "length": 1.0, "mode": "test", "epoch": 38}
{"loss": 0.6562904715538025, "acc": 0.6284232139587402, "length": 1.0002363920211792, "mode": "train", "epoch": 39}
{"loss": 0.6604598760604858, "acc": 0.630859375, "length": 1.0, "mode": "test", "epoch": 39}
{"loss": 0.6560454368591309, "acc": 0.6285956501960754, "length": 1.000247836112976, "mode": "train", "epoch": 40}
{"loss": 0.6603244543075562, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 40}
{"loss": 0.6573053598403931, "acc": 0.6283082365989685, "length": 1.000417709350586, "mode": "train", "epoch": 41}
{"loss": 0.6607515215873718, "acc": 0.6304687857627869, "length": 1.0, "mode": "test", "epoch": 41}
{"loss": 0.6562768220901489, "acc": 0.6288256049156189, "length": 1.0002305507659912, "mode": "train", "epoch": 42}
{"loss": 0.6612231135368347, "acc": 0.6306641101837158, "length": 1.0, "mode": "test", "epoch": 42}
{"loss": 0.6561516523361206, "acc": 0.6283082365989685, "length": 1.000180959701538, "mode": "train", "epoch": 43}
{"loss": 0.6598708629608154, "acc": 0.6302733421325684, "length": 1.0, "mode": "test", "epoch": 43}
{"loss": 0.6566506028175354, "acc": 0.6284233927726746, "length": 1.0001587867736816, "mode": "train", "epoch": 44}
{"loss": 0.6612604856491089, "acc": 0.6304686665534973, "length": 1.0, "mode": "test", "epoch": 44}
{"loss": 0.656190037727356, "acc": 0.6286534667015076, "length": 1.0001354217529297, "mode": "train", "epoch": 45}
{"loss": 0.6613030433654785, "acc": 0.6300780773162842, "length": 1.0, "mode": "test", "epoch": 45}
{"loss": 0.6563547253608704, "acc": 0.6279630661010742, "length": 1.0001323223114014, "mode": "train", "epoch": 46}
{"loss": 0.6601076722145081, "acc": 0.630078136920929, "length": 1.0, "mode": "test", "epoch": 46}
{"loss": 0.6561846137046814, "acc": 0.6286534070968628, "length": 1.00010347366333, "mode": "train", "epoch": 47}
{"loss": 0.6616439819335938, "acc": 0.6292968392372131, "length": 1.0, "mode": "test", "epoch": 47}
{"loss": 0.6556234359741211, "acc": 0.6287685036659241, "length": 1.000152826309204, "mode": "train", "epoch": 48}
{"loss": 0.6611272096633911, "acc": 0.6302733421325684, "length": 1.0, "mode": "test", "epoch": 48}
{"loss": 0.6563106775283813, "acc": 0.6277903914451599, "length": 1.0000746250152588, "mode": "train", "epoch": 49}
{"loss": 0.6601949334144592, "acc": 0.6289061903953552, "length": 1.0, "mode": "test", "epoch": 49}
{"loss": 0.6562562584877014, "acc": 0.6284231543540955, "length": 1.000108242034912, "mode": "train", "epoch": 50}
{"loss": 0.6630153059959412, "acc": 0.629101574420929, "length": 1.0, "mode": "test", "epoch": 50}
{"loss": 0.655677318572998, "acc": 0.6285960078239441, "length": 1.000072717666626, "mode": "train", "epoch": 51}
{"loss": 0.6608290672302246, "acc": 0.6302734017372131, "length": 1.0, "mode": "test", "epoch": 51}
{"loss": 0.6558355689048767, "acc": 0.6285386085510254, "length": 1.0000662803649902, "mode": "train", "epoch": 52}
{"loss": 0.6606351137161255, "acc": 0.6302734017372131, "length": 1.0, "mode": "test", "epoch": 52}
{"loss": 0.656300961971283, "acc": 0.6285959482192993, "length": 1.0000883340835571, "mode": "train", "epoch": 53}
{"loss": 0.6613979339599609, "acc": 0.6302734017372131, "length": 1.0, "mode": "test", "epoch": 53}
{"loss": 0.6556864976882935, "acc": 0.6286535263061523, "length": 1.0000580549240112, "mode": "train", "epoch": 54}
{"loss": 0.6600064039230347, "acc": 0.6302734017372131, "length": 1.0, "mode": "test", "epoch": 54}
{"loss": 0.6555902361869812, "acc": 0.6281933784484863, "length": 1.0000451803207397, "mode": "train", "epoch": 55}
{"loss": 0.6605931520462036, "acc": 0.6300780773162842, "length": 1.0, "mode": "test", "epoch": 55}
{"loss": 0.6559956669807434, "acc": 0.6285385489463806, "length": 1.000036597251892, "mode": "train", "epoch": 56}
{"loss": 0.6624437570571899, "acc": 0.6287108063697815, "length": 1.0, "mode": "test", "epoch": 56}
{"loss": 0.6557903289794922, "acc": 0.628135621547699, "length": 1.0000505447387695, "mode": "train", "epoch": 57}
{"loss": 0.6608895063400269, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 57}
{"loss": 0.6563112139701843, "acc": 0.6278480887413025, "length": 1.0000678300857544, "mode": "train", "epoch": 58}
{"loss": 0.6601513624191284, "acc": 0.6300780773162842, "length": 1.0, "mode": "test", "epoch": 58}
{"loss": 0.6559943556785583, "acc": 0.6281933784484863, "length": 1.000028133392334, "mode": "train", "epoch": 59}
{"loss": 0.6624557971954346, "acc": 0.6302734017372131, "length": 1.0, "mode": "test", "epoch": 59}
{"loss": 0.6563148498535156, "acc": 0.6288262009620667, "length": 1.0000249147415161, "mode": "train", "epoch": 60}
{"loss": 0.6621129512786865, "acc": 0.6302734017372131, "length": 1.0, "mode": "test", "epoch": 60}
test accuracy: 0.6323213577270508
